Wikipedia/Code/EditorViewController.swift

```diff
@@ -79,6 +80,10 @@ final class EditorViewController: UIViewController, WMFNavigationBarConfiguring
         return spinner
     }()
     
+    internal var authManager: WMFAuthenticationManager {
```

Comment:
Doesn't look like this is used anymore, but I you'll probably want it for future tasks so I'm good with leaving it here.